velox/vector/VectorStream.h

```diff
@@ -43,6 +43,15 @@ class VectorSerializer {
   /// Serialize all rows in a vector.
   void append(const RowVectorPtr& vector);
 
+  // Usage
```

Comment:
It is helpful to include usage in the documentation. Can we also provide the documentation itselt?

```
Returns the serialized size of the data previously added via 'append' methods. Can be used to allocate buffer of exact size before calling 'flush'.
```

Can this method be 'const'?